### DISCUSSION TITLE
New icon, Korean, and bug fixes

### DISCUSSION THREAD

**Discussion by Epicpkmn11** [Universal-Team/Universal-Updater#93] [Announcements]
### What's New?
- Adds Korean and most other translations have been fully completed
- Uninstalling an app now makes it not show an icon for updates
- Changed to a new icon with better shading

### Bug fixes
- Fixes fonts being too large on Chinese, Taiwanese, and Korean consoles 
- Blocks going to the HOME menu and quitting the app while the queue is running, which could cause buggy behavior
- Adds safety checks for low SD card space

### Other notes
A fairly minor update this time, but we've got enough bug fixes and new little things that we figured it was a good time.

Find any bugs we missed, have suggestions, or need help? Make an issue or discussion here on GitHub, normally we'd suggest joining our [Discord server](https://universal-team.net/discord) here too, but as of making this we're not allowing new members unless you know someone in it and have them let us know to let you in.

We hope you enjoy the new update!
~ Universal-Team

<hr /><em>This discussion was created from the release <a href='https://github.com/Universal-Team/Universal-Updater/releases/tag/v3.2.2'>New icon, Korean, and bug fixes</a>.</em>

**Comment by LinUwUxCat** [comment]
Poggers
